src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -211,6 +229,12 @@ default BootstrapMethodInvocation lookupBootstrapMethodInvocation(int index, int
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the BootstrapMethodInvocation instances for all invokedynamic
```

Comment:
Point out that the returned list is unmodifiable (like the API for `Stream.toList()` does).